css/style.css

```diff
@@ -463,6 +463,11 @@ pre {
   @media (min-width: 1200px) {
     .container {
       max-width: 1140px; } }
+      @media (min-width:641px   ) {
```

Comment:
@Ugerpaul17 ,

Nice work.
What's this css change for?

Reply:
I was tasked to look into the Alignment of logos is slightly off on 14 inches.... so I added that CSS for the 14-inch screen

Reply:
The change you added is going to affect anything else that has that tag. It's better if you add an id to where the logs are and use that id to change the appearance.

Also, it is better if you separate the changes asked for into different pull requests. Because as can be seen, the ug feature is ok, but the css is going to make it get held back.

If you can remove the css from this PR (make a commit and push the removal) and raise it in another one, that will be great. If you're able to quickly work out the css feature though, then please go ahead.

Reply:
@Ugerpaul17 , you'll need to pull this branch first. Then add the commit, if that's the path you choose

Reply:
alright, let me remove the CSS from this PR, I will probably raise a new PR for the CSS fix.

Reply:
I removed the CSS from this PR.